gap/PackageManager.gd

```diff
@@ -235,7 +235,8 @@ PKGMAN_DownloadCmds := [ [ "wget", ["--quiet", "-O", "-"] ],
 PKGMAN_CurlIntReqVer :=
   First(PackageInfo("PackageManager")[1].Dependencies.SuggestedOtherPackages,
         item -> item[1] = "curlInterface")[2];
-PKGMAN_BuildPackagesScript := Filename(List(GAPInfo.RootPaths, Directory),
-                                       "bin/BuildPackages.sh");
+PKGMAN_BuildPackagesScript := Filename(DirectoriesLibrary("bin"),
+                                       "BuildPackages.sh");
+PKGMAN_Sysinfo := Filename(DirectoriesLibrary(""), "sysinfo.gap");
```

Comment:
Actually this doesn't work either (I messed up my local test). I'll have to think about a good way to take care of this.

Reply:
Ah, it works if the dir containing sysinfo.gap was added as a GAP root. Which it need not, but in my test I had it...

Reply:
I think what we ultimately need is a function in `gap` which allows to query the path to `sysinfo.gap`.